src/quick-lint-js/parse.h

```diff
@@ -3026,6 +3026,23 @@ class parser {
               });
           goto initialize_variable;
 
+        case token_type::kw_typeof:
+        case token_type::kw_class:
+        case token_type::kw_new: {
```

Comment:
Nit: Sort cases alphabetically.
```suggestion
        case token_type::kw_class:
        case token_type::kw_new:
        case token_type::kw_typeof: {
```